docs/architecture/adr-071-proposer-based-timestamps.md

```diff
@@ -250,84 +163,55 @@ type TimestampParams struct {
 }
 ```
 
-### Changes to `Header`
+### Changes to the block proposal step
+
+#### Proposer selects block timestamp
 
-The [Header](https://github.com/tendermint/tendermint/blob/a419f4df76fe4aed668a6c74696deabb9fe73211/types/block.go#L338) struct currently contains a timestamp.
-This timestamp is set as the `BFTtime` derived from the block's `LastCommit.CommitSig` timestamps.
-This timestamp will no longer be derived from the `LastCommit.CommitSig` timestamps and will instead be included directly into the block's `LastCommit`.
-This timestamp will therfore be identical in both the `Header` and the `LastCommit`.
-To clarify that the timestamp in the header corresponds to the `LastCommit`'s time, we will rename this timestamp field to `last_timestamp`.
+The [proposal logic currently](https://github.com/tendermint/tendermint/blob/68ca65f5d79905abd55ea999536b1a3685f9f19d/internal/state/state.go#L269) sets the time in the proposed timestamp in the block `Header` as the weighted median of the times in the `LastCommit.CommitSig`s per the `BFTTime` algorithm. 
+This proposal logic will be updated to instead use the current Unix time known to the proposer as the value for the timestamp value in the `Header`.
```

Comment:
Not sure whether here we should reinforce that the timestamp in the `Header` of block `k` should be larger than the timestamp in the `Header` of block `k-1`.